### PR TITLE
Microverse: Capstone Project - Exhibition Page

### PR DESCRIPTION
**Pull request by nicupop729**
# Microverse: Capstone Project - Exhibition Page

### - In this branch were build 2 pages:

               - The home page;
               - The about page;
      
### - Each of these pages has versions for 2 different screen sizes:

                - Mobile: up to 768px wide;
                - Desktop: 768px or wider;
       
### - Interactions

                 - Links:
                            - The home page has a link in the menu to the about page;
                            - The about page has a link in the menu to the home page;
                            - The logo in the header links to the home page;
               
                 - Mobile menu:
                            - When the user clicks (or taps) the hamburger button on the
                              header, the mobile menu appears over the page.

### - Dynamic page:
                 - The section "Exhibition Sections" is created dynamically in JavaScript.
                 - It was used a JavaScript variable with the data about the exhibition
                   sections and it is used when the page loads to create the HTML for this section dynamically.

### Link to presentation of the project: https://www.loom.com/share/344b7c1988f54badb448aa5c1e56715e

### CODE REVIEW

**Comment by nicupop729** [comment]
Taking into account that the theme of this project is about Ancient Egypt, I used this specific background image and colours.
Apart from that, I used the indicated colours  
Please, consider this when reviewing the project!
Thank you very much!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            </li>
+          </ul>
+          <button type="submit" class="main-btn">Get a ticket</button>
+          <a class="see-program" href="#">See the whole program</a>
+        </section>
+        <section class="exhib-sections">
+          <h3 class="title">Exhibition Sections</h3>
+          <hr class="big-title-bar" />
+          <ul class="exhib-section-list"></ul>
+        </section>
```
Comment (rloterh) [inline index.html:121]: - [ ] The contents within the items of your main section are **not vertically aligned**, as can be seen in the original design. Kindly improve upon this aspect by ensuring that the contents within the items are vertically centered by ensuring that there is even space at both the top and bottom. 

**Your main program section:**
image


**The original main program section:**
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      </main>
+      <footer class="footer">
```
Comment (rloterh) [inline index.html:123]: The **background color** of your footer on both the mobile and desktop versions are the same. As specified let's ensure that:
- [ ]  The footer, on mobile size should have a white background, which is implemented.
- [ ] The footer, on desktop size should have a dark background.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  navDiv.style.display = 'none';
+});
+
+humbergerButton.addEventListener('click', () => {
+  document.body.appendChild(navDiv);
+  navDiv.style.backgroundColor = '#272a31';
+  navDiv.style.display = 'block';
+});
+
+// Populate dynamically the Exhibition Sections Section
```
Comment (rloterh) [inline main.js:58]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

File: README.md [hunk 4]
```diff
@@ -1,76 +1,43 @@
...
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
```
Comment (rloterh) [inline README.md:68]: - [ ] It's a nice and smart thing to give appropriate credit to **Cindy Shin**, the author of [the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Nicu, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Hosted project on Github pages is working correctly
- Good presentation of project
- Effectively implemented HTML semantics
- Mobile menu is working as intended
- The dynamic aspect is well implemented
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+            <li class="inner-nav-parent">
+              <ul class="inner-nav-list">
+                <li>
+                  <a class="menu-btn" href="about.html">About</a>
+                </li>
+                <li><a class="menu-btn" href="#">Program</a></li>
+                <li><a class="menu-btn" href="#">Forum</a></li>
+                <li><a class="menu-btn" href="#">Sponsor</a></li>
+                <li><a class="menu-btn" href="#">News</a></li>
+              </ul>
```
[the remark this answers is not in the document]
Comment (nicupop729) [reply index.html:45]: https://www.loom.com/share/63e98eb34d7348dd8b32b1339a987c0a

**Pull request merged**